src/wallet/wallet.h

```diff
@@ -533,6 +533,11 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+enum CreateTransactionFlags {
+    CREATE_TX_DEFAULT     = 0,
```

Comment:
It seems to me default is simply "not bip125". Not sure the name "default" really fits, especially longer-term.